src/Cookie/CookieWrapper.php

```diff
@@ -53,9 +52,6 @@ public function set(string $name, ?string $value = NULL, array $options = []): v
         }
 
         $path = $options['path'] ?? '/';
-        if ($this->has_ssl_available and ($options['samesite'] ?? NULL)) {
```

Comment:
As we can now pass `samesite` through in the options (if present) I think all we need to do is` if (!$this->has_ssl_available) { unset $options['samesite'];} `